internals/review_models.py

```diff
@@ -137,7 +137,7 @@ class Comment(ndb.Model):
   """A review comment on a feature."""
   feature_id = ndb.IntegerProperty(required=True)
   field_id = ndb.IntegerProperty()  # The approval field_id, or general comment.
-  created = ndb.DateTimeProperty(auto_now=True)
+  created = ndb.DateTimeProperty()
```

Comment:
I agree that auto_now was wrong.  Why not use auto_now_add=True?

Reply:
Yes, that's what should be used - I was not aware of the difference, but now it's been changed